package.json

```diff
@@ -85,7 +85,7 @@
     "gl-scatter3d": "^1.2.0",
     "gl-select-box": "^1.0.3",
     "gl-spikes2d": "^1.0.2",
-    "gl-streamtube3d": "^1.1.3",
+    "gl-streamtube3d": "git://github.com/gl-vis/gl-streamtube3d.git#39df57d38273f76a20627192a44e292caa003a68",
```

Comment:
https://github.com/gl-vis/gl-streamtube3d/compare/align-tubes-with-mesh-and-cone